main.js

```diff
@@ -0,0 +1,95 @@
+
+let w = 40;
+let h = 30;
+let world = Array(h);
+for(let i = 0; i < world.length; i ++){
+  world[i] = Array(w);
+  world[i].fill("[ ]");
+}
+// seed
+world[(Math.floor(h/2))][(Math.floor(w/2))] = "[0]";
+// world[(Math.floor(h/2))-1][(Math.floor(w/2))-1] = "[0]";
+world[(Math.floor(h/2))+1][(Math.floor(w/2))+1] = "[0]";
+world[(Math.floor(h/2))+1][(Math.floor(w/2))+2] = "[0]";
+world[(Math.floor(h/2))-1][(Math.floor(w/2))+1] = "[0]";
+world[(Math.floor(h/2))+1][(Math.floor(w/2))] = "[0]";
+
+let world2 = copyworld(world);
```

Comment:
Why do you need world2, and then also `copyworld`? Couldn't you do the cloning inside the copyworld? Slightly confusing. Also can be a `const`.